<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefineSetupDialog</class>
 <widget class="QDialog" name="DefineSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>183</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Define Simulation Setup</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameEdit">
        <property name="toolTip">
         <string>Name of the Setup. Used as unique identification in one environment.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fileLabel">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="fileEdit">
          <property name="toolTip">
           <string>File to save the Simulation Setup in. Must not be a file already in use by another Simulation Setup.</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="fileToolButton">
          <property name="toolTip">
           <string>Define a Simulation Setup file.</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="imageresources.qrc">
            <normaloff>:/folder</normaloff>:/folder</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>fileToolButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>fileEdit</tabstop>
 </tabstops>
 <resources>
  <include location="imageresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DefineSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DefineSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileToolButton</sender>
   <signal>clicked()</signal>
   <receiver>DefineSetupDialog</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DefineSetupDialog</receiver>
   <slot>setName(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DefineSetupDialog</receiver>
   <slot>setFile(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setName(QString)</slot>
  <slot>setCaption(QString)</slot>
  <slot>setFile(QString)</slot>
  <slot>loadFile()</slot>
 </slots>
</ui>
